blueprints/third-party-solutions/phpipam/main.tf

```diff
@@ -0,0 +1,153 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  all_principals_iam = [for k in var.principals : "user:${k}"]
+  cloudsql_conf = {
+    database_version = "MYSQL_8_0"
+    tier             = "db-g1-small"
+    db               = "phpipam"
+    user             = "admin"
+  }
+  domain = (
+    var.custom_domain != null ? var.custom_domain : (
+      var.phpipam_exposure == "EXTERNAL" ?
+    "${google_compute_global_address.default.0.address}.nip.io" : "phpipam.internal")
+  )
+  iam = {
+    # CloudSQL
+    "roles/cloudsql.admin"        = local.all_principals_iam
+    "roles/cloudsql.client"       = local.all_principals_iam
+    "roles/cloudsql.instanceUser" = local.all_principals_iam
+    # common roles
+    "roles/logging.admin"                  = local.all_principals_iam
+    "roles/iam.serviceAccountUser"         = local.all_principals_iam
+    "roles/iam.serviceAccountTokenCreator" = local.all_principals_iam
+  }
+  connector        = var.connector == null ? google_vpc_access_connector.connector.0.self_link : var.connector
+  phpipam_password = var.phpipam_password == null ? random_password.phpipam_password.result : var.phpipam_password
+  network          = var.vpc_config == null ? module.vpc.0.self_link : var.vpc_config.network
+  subnetwork       = var.vpc_config == null ? module.vpc.0.subnet_self_links["${var.region}/ilb"] : var.vpc_config.subnetwork
+}
+
+
+# either create a project or set up the given one
+module "project" {
+  source          = "../../../modules/project"
+  name            = var.project_id
+  parent          = try(var.project_create.parent, null)
+  billing_account = try(var.project_create.billing_account_id, null)
+  project_create  = var.project_create != null
+  prefix          = var.project_create == null ? null : var.prefix
+  iam             = var.project_create != null ? local.iam : {}
+  services = [
+    "run.googleapis.com",
+    "logging.googleapis.com",
+    "monitoring.googleapis.com",
+    "sqladmin.googleapis.com",
+    "sql-component.googleapis.com",
+    "vpcaccess.googleapis.com",
+    "servicenetworking.googleapis.com"
+  ]
+}
+
+
+# create a VPC for CloudSQL and ILB
+module "vpc" {
+  count      = var.vpc_config == null ? 1 : 0
+  source     = "../../../modules/net-vpc"
+  project_id = module.project.project_id
+  name       = "${var.prefix}-sql-vpc"
+  subnets = [
+    {
+      ip_cidr_range = var.ip_ranges.ilb
+      name          = "ilb"
+      region        = var.region
+    }
+  ]
+  psa_config = {
+    ranges = {
+      cloud-sql = var.ip_ranges.psa
+    }
+  }
+}
+
+
+# create a VPC connector for the ClouSQL VPC
+resource "google_vpc_access_connector" "connector" {
```

Comment:
the `cloud-run` module can [create the access connector](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/988fd2ee0548ca4ee7159a0932703df241a3fa86/modules/cloud-run/variables.tf#L232) for you

Reply:
Thanks, removed the resource and configured vpc connector for the cloud run module!